lab-katherine/src/main.js

```diff
@@ -0,0 +1,51 @@
+import './styles/main.scss';
+import React from 'react';
+import ReactDom from 'react-dom';
+const cowsay = require('cowsay-browser');
```

Comment:
this way works, but you can also import cowsay from 'cowsay-browser', and same with faker :)